src/release/program-github-release.ts

```diff
@@ -40,7 +40,7 @@ export default (parent: Command) =>
           appOctokit,
           options.owner,
           options.repo,
-          options.productionRelease,
+          options.productionDelivery,
```

Comment:
🔧 I think `productionRelease` is actually more correct than `productionDelivery` in this case as it only concerns the specific github release. Therefore, I'd prefer if we kept it this way here and changed the option in integreat-app.